src/platform/backends/lxd/lxd_virtual_machine_factory.cpp

```diff
@@ -247,7 +238,7 @@ auto mp::LXDVirtualMachineFactory::networks() const -> std::vector<NetworkInterf
                 ret.push_back(std::move(network));
 
         for (auto& net : ret)
-            if (net.needs_authorization && find_bridge_with(ret, net.id) != ret.cend())
+            if (net.needs_authorization && MP_BACKEND.find_bridge_with(ret, net.id) != ret.cend())
                 net.needs_authorization = false;
```

Comment:
Change this to match https://github.com/canonical/multipass/pull/3463/files#diff-77064bb2fbd2589b9a34679c3e6bde02dc7ad8b0b22931195d4d687e8aef2720R232? It makes it easier to see that the logic is the same in both places.

Reply:
Right, changed in the base branch.

Reply:
This still isn't changed

Reply:
Oh, sorry, I didn't get it. I changed it as you propose, but I used `&=` instead of `=` to maintain exactly the same logic (i.e., the same exact truth table).